cypress/support/userData.js

```diff
@@ -0,0 +1,11 @@
+const { faker } = require('@faker-js/faker');
+
+export const workerData = () => {
+  return {
+    firstName: faker.person.firstName(),
+    lastName: faker.person.lastName(),
+    email: faker.internet.email(),
+    salary: faker.number.int({ min: 30000, max: 100000 }),
```

Comment:
The method `faker.number.int` is not correct. The correct method to generate a random integer in a range is `faker.datatype.number`. Please update this line to use `faker.datatype.number({ min: 30000, max: 100000 })`.